python/ray/worker.py

```diff
@@ -494,10 +496,14 @@ def get_objects(self, object_ids):
                     "which is not an ray.ObjectID.".format(object_id))
 
         if self.mode == LOCAL_MODE:
+            if timeout is not None:
```

Comment:
Can you add a `TODO` here to remove this once we move local mode to the core worker?